src/utils/registration-helpers.ts

```diff
@@ -709,3 +710,11 @@ export const getDisabledCategories = (
 
   return disabledCategories;
 };
+
+export const findParentSubject = (json: NpiSubjectDomain[], npiSubjectHeadingId: string): string | null => {
```

Comment:
Mer forklarende navn her kanskje? (`npiDisciplines` er bare ett av sikkert mange mulige forslag)
```suggestion
export const findParentSubject = (npiDisciplines: NpiSubjectDomain[], npiSubjectHeadingId: string): string | null => {
```